client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -452,13 +452,7 @@ class EmailProvidersComparison extends React.Component {
 					comment:
 						'%(titanProductName) is the name of the product, which should be "Professional Email" translated',
 			  } )
-			: translate( 'Add %(titanProductName)s', {
-					args: {
-						titanProductName: getTitanProductName(),
-					},
-					comment:
-						'%(titanProductName) is the name of the product, which should be "Professional Email" translated',
-			  } );
+			: translate( 'Add Email' );
```

Comment:
Can you please use the existing code with the translated name? I believe the designs predate our rebrand to Professional Email.

Reply:
Sorry, I followed the Figma and realized that it had changed to "Add Email". I've reverted it back.